srcpkgs/libplacebo/template

```diff
@@ -1,32 +1,32 @@
 # Template file for 'libplacebo'
 pkgname=libplacebo
-version=4.208.0
+version=5.229.2
 revision=1
 build_style=meson
 configure_args="-Dshaderc=enabled -Dvulkan=enabled
- -Dlcms=$(vopt_if lcms en dis)abled -Dopengl=$(vopt_if opengl en dis)abled"
-hostmakedepends="pkg-config python3-Mako Vulkan-Headers"
-makedepends="shaderc Vulkan-Headers vulkan-loader $(vopt_if lcms lcms2-devel)
- $(vopt_if opengl libepoxy-devel)"
+ -Dlcms=$(vopt_if lcms en dis)abled -Ddemos=false -Dtests=true"
+hostmakedepends="pkg-config Vulkan-Headers python3-glad"
+makedepends="shaderc Vulkan-Headers vulkan-loader python3-MarkupSafe
+ python3-Jinja2 $(vopt_if lcms lcms2-devel)"
 short_desc="Reusable library for GPU-accelerated video/image rendering"
 maintainer="Enrico Belleri <idesmi@protonmail.com>"
 license="LGPL-2.1-or-later"
 homepage="https://code.videolan.org/videolan/libplacebo"
 distfiles="https://code.videolan.org/videolan/libplacebo/-/archive/v${version}/libplacebo-v${version}.tar.gz"
-checksum=7b3c857934ee3d30f743e43d7f0606e10950806661ea0ea385f8a1f06cbab854
+checksum=a42035c69e4a6373cb9856cc0c5736f24af4485006f3b29605ddffba377264db
 
 if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
 	makedepends+=" libatomic-devel"
 	LDFLAGS+=" -latomic"
 fi
```

Comment:
```suggestion
fi

if [ -n "$XBPS_CHECK_PKGS" ]; then
	configure_args+=" -Dtests=true"
else
	configure_args+=" -Dtests=false"
fi
```

(I like to add both branches just in case the default value of the option changes)